frontend/packages/topology/src/utils/topology-utils.ts

```diff
@@ -44,13 +44,28 @@ export const getCheDecoratorData = (consoleLinks: K8sResourceKind[]): CheDecorat
   };
 };
 
+const getFullGitURL = (gitUrl: GitUrlParse.GitUrl, branch?: string) => {
+  const baseUrl = `https://${gitUrl.source}/${gitUrl.owner}/${gitUrl.name}`;
+  if (!branch) {
+    return baseUrl;
+  }
+  switch (gitUrl.resource) {
+    case 'github.com':
+      return `${baseUrl}/tree/${branch}`;
+    case 'gitlab.com':
+      return `${baseUrl}/-/tree/${branch}`;
+    case 'bitbucket.org':
+      return `${baseUrl}/src/${branch}`;
+    default:
+      return baseUrl;
+  }
```

Comment:
To support `github.example.com` and so on I would use `gitUrl.resource.includes("gitlab")` etc. here instead. Wdyt?

Reply:
Updated.